docs/how-to/custom-env.txt

```diff
@@ -171,17 +171,19 @@ image (using the ``FROM`` instruction).
 Here is an example of a ``Dockerfile`` that installs both ``conda``- and
 ``pip``-based dependencies.
 
-.. code::
+.. code:: bash
 
    FROM determinedai/environments:cuda-10.2-pytorch-1.7-tf-1.15-gpu-0.10.0
-   RUN apt-get update && apt-get install -y unzip python-opencv graphviz
-   COPY environment.yml /tmp/environment.yml
-   COPY pip_requirements.txt /tmp/pip_requirements.txt
-   RUN conda env update --name base --file /tmp/environment.yml && \
-       conda clean --all --force-pkgs-dirs --yes
+   RUN apt-get update
```

Comment:
Nit: I think because of the way Dockerfiles skip lines that were previously successful, it's best-practice to keep apt-get update and apt-get install on the same line with the &&'s.

Reply:
Okay, updated but the line is now longer though because the tz data has to occur after the update but before graphviz.